Analysis/Tasks/PWGDQ/tableMaker.cxx

```diff
@@ -48,6 +48,9 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 using MyBarrelTracks = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksCov, aod::TracksExtended, aod::TrackSelection, aod::pidRespTPC, aod::pidRespTOF, aod::pidRespTOFbeta>;
```

Comment:
Could you already subscribe to the split TPC PID tables? Either tiny or split?

Reply:
I am planning to do this. In fact i am discussing now with Nicolo regarding the tiny version. I will ask one of the people in our group to run a check with the tiny version